data/coalition outfits.txt

```diff
@@ -90,7 +92,8 @@ outfit "Small Shield Module"
 	"shield generation" .28
 	"shield energy" .28
 	"shield heat" .1
-	description "Although combat is almost unheard of in Coalition space, almost all of their ships are able to recharge their shields when necessary."
+	"shields" 560
+	description "Although combat is almost unheard of in Coalition space, almost all of their ships are able to recharge their shields when necessary. Coalition shield rechargers make their shields more resilient to damage."
```

Comment:
Outfits that add shield capacity or hull strength are generally verboten because they would allow you to stuff any ship with enough of them to turn it into a heavy warship.

The wiki [explicitly recommends against this](https://github.com/endless-sky/endless-sky/wiki/CreatingOutfits):

> I recommend against providing outfits that give ships additional shield strength, because that could create balancing issues. But if you want to do it, this is the attribute to modify.

Any outfit that increased shield strength or hull capacity would need to incur its ship significant drawbacks to even be considered.